README.md

```diff
@@ -891,6 +891,10 @@ class Employee extends Person {
 }
 ```
 
+## Supporting Development
+
+We are looking for partner companies to help us evolve Microstates. If you are interested in supporting Microstates development by hiring Frontside, please reach out via [email](mailto:info@frontside.io)
```

Comment:
Should this be `info@`? Or `oss@`, `sales@`?

Reply:
Sales?

Reply:
What's the call here?

Reply:
I think any of it works. `oss@frontside.io` makes the most sense imo